src/rgw/rgw_data_sync.cc

```diff
@@ -2616,12 +2616,6 @@ class RGWBucketSyncSingleEntryCR : public RGWCoroutine {
             retcode = -EIO;
           } else if (op == CLS_RGW_OP_ADD ||
                      op == CLS_RGW_OP_LINK_OLH) {
-            if (op == CLS_RGW_OP_ADD && !key.instance.empty() && key.instance != "null") {
```

Comment:
a simple way is judge whether version op,  as add "&& versioned", it will sync the last acl op

Reply:
thanks, you're right